config/pwa/amazon_music.desktop

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env xdg-open
+[Desktop Entry]
+Version=1.0
+Terminal=false
+Type=Application
+Name=Amazon Music
+Exec=google-chrome-stable "--profile-directory=Profile 1" --app-id=dojpeppajphepagdhclblkkjnoaeamee
```

Comment:
https://superuser.com/questions/1648930/how-can-i-change-the-chrome-profile-numbers